ansible/roles/elasticsearch/templates/config/elasticsearch.yml.j2

```diff
@@ -9,6 +9,11 @@ node.name: "{{ elasticsearch_node_name }}"
 
 transport.tcp.port: {{ elasticsearch_tcp_port }}
 
+{% if elasticsearch_bind_to_localhost %}
```

Comment:
i think a better way to do this check is to check the length of `group['all']` and if it's greater than 1 than it's not a monolith. that way the person ansible deploying doesn't have to know about this variable to do a correct deploy